homeassistant/components/isy994/services.py

```diff
@@ -230,18 +230,3 @@ async def _async_rename_node(call: ServiceCall) -> None:
         schema=cv.make_entity_service_schema(SERVICE_RENAME_NODE_SCHEMA),
         service_func=_async_rename_node,
     )
-
-
-@callback
-def async_unload_services(hass: HomeAssistant) -> None:
```

Comment:
In a similar way, I think the code would be simplified in `isy994` if all the methods were moved to be module methods, and `service.hass` property was used instead.

Reply:
existing_services check can also be removed in `async_setup_services`